workflows/SSD_index.md

```diff
@@ -0,0 +1,62 @@
+**Usage for SSD-based indices**
+===============================
+
+To generate an SSD-friendly index, use the `tests/build_disk_index` program. 
+----------------------------------------------------------------------------
+
+```
+./tests/build_disk_index  data_type<float/int8/uint8>   dist_fn<l2/mips>   data_file.bin   index_prefix_path   R(graph degree)   L(build complexity)   B(search memory allocation in GB)   M(build memory allocation in GB)   T(#threads)   PQ_disk_bytes" 
+```
+
+The arguments are as follows:
+
+(i) **data_type**: The type of dataset you wish to build an index. float(32 bit), signed int8 and unsigned uint8 are supported. 
```

Comment:
In the memory index I mentioned the markdown for lists - just wanted to reference it here too https://github.com/microsoft/DiskANN/pull/48/files#r824174894